### ISSUE TITLE
Initial Vertical Split

### ISSUE THREAD

**Issue by ghosh** [eggplanetio/mert#4]
Thanks for making this. Finally I can stop using tmux to just use split screens.

However, I am having difficulty in setting it up. For e.g: - How would i be able to make it look like this?

```
+---------------------|---------------------+
| "Row 1, Pane 1"     | "Row 1, Pane 2      |
|                     |                     |
|                     |---------------------|
|                     | "Row 2, Pane 2      |
|                     |                     |
+---------------------|---------------------+
```

I tried the following, but it didn't work

```
layout:
  -
    - echo "Row 1, Pane 1 in bar"
    -
      - echo "Row 1, Pane 2"
      - echo "Row 2, Pane 2"
```

**Comment by briangonzalez** [comment]
We _could_ achieve this by changing the command `mert` uses to split. In essence, we'd instead use `commands.SPLITPANE` instead. 

See this line of code: https://github.com/eggplanetio/mert/blob/master/lib/launcher.js#L40

Better idea: let's make this configurable.

I was thinking we could define another key in the yaml called `split_strategy` which would accept `horizontal | vertical` and would default to `vertical` (today's behavior).

With this `.mertrc` and a small code code, we could achieve what you desire.

``` yaml
split_strategy: "horizontal"
layout:
  -
    - 
      - echo "Row 1, Pane 1 in bar"
    -
      - echo "Row 1, Pane 2"
      - echo "Row 2, Pane 2"
```

**Comment by briangonzalez** [comment]
@Ghosh Want to take a stab and send over a PR?

**Comment by briangonzalez** [comment]
@juancaldera Thoughts?

**Comment by ghosh** [comment]
@briangonzalez Great. Sounds good. Thanks for the explanation, I'll give it a go and submit a pull.

**Comment by briangonzalez** [comment]
Fixed in #7. Thank you!
